src/openrct2-ui/windows/EditorObjectSelection.cpp

```diff
@@ -599,12 +599,14 @@ class EditorObjectSelectionWindow final : public Window
             if (!objectSelectResult.Successful)
                 return;
 
-            // Close any other open windows such as options/colour schemes to prevent a crash.
-            WindowCloseAll();
-            // WindowClose(*w);
+            auto& objRepository = OpenRCT2::GetContext()->GetObjectRepository();
```

Comment:
have you checked the interaction with the options/colour schemes windows if they are open at this point?